repos.yaml

```diff
@@ -683,6 +683,11 @@ repositories:
     maintainers:
       - email: shih5@chinaunicom.cn
         name: Hao Shi
+  - name: gp42
+    url: https://ops42.org
```

Comment:
The URL in both files should be the same and to the charts repository. Can you update this to be `https://ops42.org/helm-charts`.

Reply:
updated